doc/book/quick-start-skeleton.md

```diff
@@ -0,0 +1,259 @@
+# Quick Start: Using the Skeleton + Installer
+
+The easiest way to get started with Expressive is to use the [skeleton
+application and installer](https://github.com/zendframework/zend-expressive-skeleton).
+The skeleton provides a generic structure for creating your applications, and
+prompts you to choose a router, dependency injection container, template
+renderer, and error handler from the outset.
+
+## 1. Create a new project
+
+First, we'll create a new project, using Composer's `create-project` command:
+
+```bash
+$ composer create-project zendframework/zend-expressive-skeleton expressive
+```
+
+This will prompt you to choose:
+
+- A router. We recommend using the default, FastRoute.
+- A dependency injection container. We recommend using the default, Zend
+  ServiceManager.
+- A template renderer. You can ignore this when creating an API project, but if
+  you will be creating any HTML pages, we recommend installing one. We prefer
+  Plates.
+- An error handler. Whoops is a very nice option for development, as it gives
+  you extensive, browseable information for exceptions and errors raised.
+
+## 2. Start a web server
+
+The Skeleton + Installer creates a full application structure that's ready-to-go
+when complete. You can test it out using [built-in web
+server](http://php.net/manual/en/features.commandline.webserver.php).
+
+From the project root directory, execute the following:
+
+```bash
+$ php -S 0.0.0.0:8080 -t public/
+```
+
+This starts up a web server on localhost port 8080; browse to
+http://localhost:8080/ to see if your application responds correctly!
+
+## Next Steps
+
+The skeleton makes the assumption that you will be writing your middleware as
+objects, and using configuration to map routes to middleware. It also provides a
+default structure for templates, if you choose to use them. Let's see how you
+can create first vanilla middleware, and then templated middleware.
+
+### Creating middleware
+
+The easiest way to create middleware is to create a class that defines an
+`__invoke()` method accepting a request, response, and callable "next" argument
+(for invoking the "next" middleware in the queue). The skeleton defines an `App`
+namespace for you, and suggests placing middleware under the namespace
+`App\Action`.
+
+Let's create a "Hello" action. Place the following in
+`src/Action/HelloAction.php`:
+
+```php
+<?php
+namespace App\Action;
+
+class HelloAction
+{
+    public function __invoke($request, $response, callable $next)
```

Comment:
As it's an example everyone will follow as good practice, I think we should typehint PSR interfaces here.